### PR TITLE
DOC: Contribution Guide

### PR DESCRIPTION
**Pull request by ostanley**
## Proposed changes

Initial commit of contributor guide for the project. Issue #96 

TODOs: recognition document

## Types of changes

What types of changes does your code introduce? Put an `x` in the boxes that apply

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionalitiy)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (if none of the other choices apply)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you are unsure about any of the choices, don't hesitate to ask!

- [ ] Changes have been tested to ensure that fix is effective or that a feature works.
- [ ] Changes pass the unit tests
- [x] I have included necessary documentation or comments (as necessary)
- [ ] Any dependent changes have been merged and published

## Notes
All PRs will undergo the unit testing before being reviewed. You may be requested to explain or make additional changes before the PR is accepted.

_PR template was adopted from [appium](https://github.com/appium/appium/blob/master/.github/PULL_REQUEST_TEMPLATE.md)_

### CODE REVIEW

**Comment by ostanley** [comment]
Added PR labels and our names. Did alphabetical last but I'd love if someone else made that call ;) 

Ready to pull in whenever you guys want.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Introduction
+
+Contributors (alphabetical last):  Greydon Gilmore, Jason Kai, Tristan Khuen, Jonathan Lau, Patrick Park, Jak Loree-Spacek, Olivia Stanley
```
Comment (kaitj) [inline CONTRIBUTING.md:3]: Tristan's last name is misspelt here - "Kuehn"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+_If the migrations folder has been changed, run `python manage.py db upgrade`_
+3. Push to git (`git push -u origin feature-branch`) and create a PR if needed
+4. Push to heroku `git push heroku master`
+
+### How to label issues
+Please use the following labels to help organize your issues and PRs.
+
+API: an (incompatible) API change\
+BENCH: changes to the benchmark suite\
+BLD: change related to building numpy\
```
Comment (kaitj) [inline CONTRIBUTING.md:40]: This should probably also mention AFIDs validator

**Review by kaitj — CHANGES_REQUESTED** [review]
Looks good. Made a couple of comments to be changed. 

Can we also add code should be run through `black` before PR / commit (in particular `black` with the `-l 79` flag, which ensures the line width still follows PEP8)? That is what Tristan used for refactor / linting, and I think it would be good to just be consistent there. Open to others (eg. `autopep8`) if the consensus is we should use that instead.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ostanley) [reply CONTRIBUTING.md]: Sorry Tristan, I fixed it.

**Comment by ostanley** [comment]
Addressed changes

**Comment by kaitj** [comment]
Looks good! Merging. Thanks @ostanley

**Pull request merged**